service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/bucket/SetReferencedDestinationGcsBucketResponseStep.java

```diff
@@ -67,7 +67,7 @@ public StepResult doStep(FlightContext context) throws InterruptedException, Ret
 
     final ApiClonedControlledGcpGcsBucket apiClonedBucket =
         new ApiClonedControlledGcpGcsBucket()
-            .effectiveCloningInstructions(ApiCloningInstructionsEnum.REFERENCE)
```

Comment:
why is it always copy_reference here?

Reply:
Good catch! That needs the resolved cloning instructions